docs/content/tutorial/saving-your-data.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: "Tutorial, part six: Saving your data | Dagster Docs"
+description: Learn how to use I/O managers to save your data.
+---
+
+# Tutorial, part six: Saving your data
+
+You've written a data pipeline using assets that runs every hour. Currently, your assets are stored in a temporary folder that is deleted after closing the `dagster` process. **Note**: If you set up Dagster without the tutorial, you may have set a `DAGSTER_HOME` environment variable. If so, then your assets will be stored in that directory.
+
+You likely want to store your assets in places such as tables in your database or files in cloud storage. To do this, Dagster has I/O managers that enable reading and writing data to storage systems.
+
+By the end of this section, you will:
+
+- Store your files in a file system
+- Store your data frames as tables in a database
+- Learn about I/O managers
+
+---
+
+## What are I/O managers?
+
+I/O stands for **input** and **output.** I/O managers are Dagster objects that control how Dagster reads and writes data to specific external services, such as Snowflake or Amazon Web Services (AWS) S3.
+
+They manage **input** by reading an asset from where it’s stored and loading it into memory to be used by a dependent asset. For example, `topstories_word_cloud` needs `topstories`. Therefore, the I/O manager that `topstories` uses will load the data into memory so the `topstories_word_cloud` asset can use it.
+
+I/O managers control \*_output_ by writing the asset's to the location configured. By the end of this section, you’ll configure an I/O manager to write the `hackernews_topstories` DataFrame into a database as a table.
+
+<Note>
```

Comment:
My instinct would be to leave this out, because it's kind of comparing apples to orange - in-memory _processing_ vs. on-disk _storage_. The main reason we load data into memory isn't because it's faster, it's because computers can't run instructions on data that's on disk (other than loading that data into memory).

If someone is skimming this, they might get the misconception that, if they're not using an IO manager, their data is stored in memory.